tests/test_jax_nonbonded.py

```diff
@@ -47,16 +47,16 @@
 def resolve_clashes(x0, box0, min_dist=0.1):
     def urt(x, box):
         distance_matrix = distance(x, box)
-        i, j = np.triu_indices(len(distance_matrix), k=1)
+        i, j = jnp.triu_indices(len(distance_matrix), k=1)
```

Comment:
not important for merging: but i'm surprised that these used jnp!

Reply:
Agreed!

Will prefer to address this in a separate PR so this one can be purely mechanical / added to `.git-blame-ignore-revs`